src/onegov/org/cli.py

```diff
@@ -884,3 +890,65 @@ def mark_as_published(request: 'OrgRequest', app: 'OrgApp') -> None:
             )
 
     return mark_as_published
+
+
+@cli.command(name="delete-invisible-links")
+def delete_invisible_links() -> 'Callable[[OrgRequest, OrgApp], None]':
+    """ Deletes all the data associated with a period, including:
+    Example:
+        onegov-org --select /foo/bar delete-invisible-links
+    """
+
+    def delete_invisible_links(request: 'OrgRequest', app: 'OrgApp') -> None:
+        session = request.session
+        query = QueryChain(
+            (session.query(Page),
+             session.query(DaypassResource),
+             session.query(ItemResource),
+             session.query(RoomResource),
```

Comment:
```suggestion
             session.query(Resource),
```
Just like with `Page` you can simplify this down to a single query using the common base class.